docs/documentation/development/bindings/xml-reference.md

```diff
@@ -279,6 +279,10 @@ Bridge and *Thing* descriptions must be placed as XML file(s) (with the ending `
 
     <channels>
       <channel id="channelID" typeId="channelTypeID" />
+      <channel id="channelID" typeId="channelTypeID">
```

Comment:
I'd suggest adding an "OR" above here as this is how it is done in the other places here as well.